docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md

```diff
@@ -0,0 +1,27 @@
+# Database Configuration
+
+## Creating the Database
+MySQL or MariaDB must be installed and a `root` or admin-level user 
+credential must be created before continuing. (This is not the same as a unix 
+        `root` credential.) 
+
+After running the install script, you will be prompted to visit the 
+`installdb.php` page at the IP address that LORIS is hosted at.
```

Comment:
It should be at the canonical domain that it's to be hosted at, the install script uses the place that it's
accessed from in order to derive `$host`